src/python/dxpy/bindings/apollo/assay_filtering_json_schemas.py

```diff
@@ -9,7 +9,7 @@
     },
     "expression": {
         "type": dict,
-        "properties": {"min_value": {"type": str}, "max_value": {"type": str}},
+        "properties": {"min_value": {"type": str,"required":False}, "max_value": {"type": str,"required":False  }},
```

Comment:
Were you seeing any strange behavior when this was not part of the schema? If so, I think there might be a bug somewhere in the class code which I should fix.

Reply:
I think I just assumed that those were meant to be required, but I'm realizing that they probably are not meant to be required.  I'll undo this change